### DISCUSSION TITLE
Does the scraper work w/twitter and gmail accounts?

### DISCUSSION THREAD

**Discussion by 7zed** [UltimaHoarder/UltimaScraper#1170] [Q&A]
just wondering.

**Comment by M-rcus** [comment]
No.

You can use [gallery-dl](https://github.com/mikf/gallery-dl) instead, which supports both Twitter and reddit - among [a long list of other websites](https://github.com/mikf/gallery-dl/blob/master/docs/supportedsites.md#supported-sites)

**Comment by LunarPenguin** [comment]
Im not sure what the question is. If you want to now if you can use the script if you login to OF with your twitter or gmail account than i think the answer is yes (for twitter at least not sure about gmail, havent tested it), i use twitter login personally, just login normally and when you are on the OF homepage open the inspector and follow the instruction in the `README.MD` file on the homepage of this repo.
